lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -141,7 +141,10 @@
     - ' - C(autoselect_datastore) (bool): select the less used datastore. Specify only if C(datastore) is not specified.'
   cdrom:
     description:
-    - A CD-ROM configuration for the VM.
+    - A list of CD-ROMs to configure.
```

Comment:
Add a note saying that - `cdrom` accepts a list of CD-ROM configurations from version 2.6